.github/workflows/cmake-multi-platform.yml

```diff
@@ -65,6 +65,14 @@ jobs:
       if: ${{ matrix.os == 'windows-latest' }}
       run: nuget install boost -Version 1.84.0
 
+    - name: Install OPC-UA
+      run: |
+        git clone https://github.com/open62541/open62541 -b v1.3.4
```

Comment:
this should definitely be a newer version, see for example the recent update in 4diac Build Environment: https://github.com/eclipse-4diac/4diac-fbe/commit/424637726f2a924cac643ae824fe174bd03565e5

Reply:
I think it would make sense to extract the build procedures of other libraries into separate yml files/functions as the workflow gets cluttered already with all the libraries.
But I must admiti didn't take a look on how to do that